providers/genericparser/base.go

```diff
@@ -44,35 +44,31 @@ func (_ GenericParser) FromURL(selector, url string) (*money.Money, error) {
 	if t == "" {
 		return nil, fmt.Errorf("selector didn't match anything")
 	}
-	t = strings.TrimFunc(t, unicode.IsSpace)
-	t = strings.ReplaceAll(t, "\n", "")
 
 	currency := "TRY" // assume default
-	if strings.HasSuffix(t, "TL") {
+	if regexp.MustCompile(`\b(tl|try|₺)\b`).MatchString(strings.ToLower(t)) {
```

Comment:
This prob won't match TL uppercase?

Reply:
Before matching process string gets through strings.ToLower function Also, unit tests include uppercase TL